app/script/util/Environment.js

```diff
@@ -45,6 +45,13 @@ window.z.util = z.util || {};
   };
 
   const _check = {
+    get_electron_version: function(user_agent) {
```

Comment:
This is not the electron version but the Wire version. I would name it desktop_version